sys-process/pm2/pm2-5.3.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit nodejs-mod systemd
+
+DESCRIPTION="Process manager for Node.js applications with a built-in load balancer"
+HOMEPAGE="https://pm2.keymetrics.io/"
+SRC_URI="https://github.com/Unitech/pm2/archive/${PV}.tar.gz -> ${P}.tar.gz
+			https://raw.githubusercontent.com/inode64/inode64-overlay/main/dist/${P}-node_modules.tar.xz"
+
+LICENSE="AGPL-3"
```

Comment:
You should include the licence of all used modules too.

Reply:
> I would have to create an application similar to https://github.com/mitchellh/golicense to extract the licenses, I had not thought of that.
> 
> Thank you

`find node_modules -type f -name package.json -exec grep '"license":' {} +`